### PR TITLE
`General`: Shrink course icon size in course header

### PR DESCRIPTION
**Pull request by JohannesStoehr**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-tests/).
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context

Improve the header further by reducing unnecessary waste of screen space.

### Description

Most often the course icon was larger than the course title followed by the description. So the course icon is shrunken down a bit to better fit with common course descriptions, so the header takes up less space.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to a Course with an icon
3. Look at the header

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Screenshots

Bildschirm­foto 2023-01-13 um 00 59 27

### CODE REVIEW

**Review by grochlitzzz — COMMENTED** [review]
In comparison with a Course without an icon, the header is still a bit too large in my opinion, maybe it could be shrunk a bit more.

image
image

**Review by PhilipWerz — DISMISSED** [review]
Course Icon size is better now. One could argue that it is still a bit too big. But I personally think that it is good the way it is now. Tested on TS1.

**Review by timo07k — DISMISSED** [review]
Works as expected. Might be personal preference, but it could still be a bit smaller for me. Tested on TS1.

**Comment by JohannesStoehr** [comment]
I talked to Stephan about potentially reducing the icon even more, but decided against it.
But we further reduced the blank boarder around it

**Review by ge65cer — APPROVED** [review]
LGTM

File: src/main/webapp/app/overview/header-course.component.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
-<div class="course-info-bar" [ngStyle]="{ backgroundColor: course.color || ARTEMIS_DEFAULT_COLOR }" id="course-detail-info-bar">
+<div class="course-info-bar course-info-bar--header" [ngStyle]="{ backgroundColor: course.color || ARTEMIS_DEFAULT_COLOR }" id="course-detail-info-bar">
     <div class="row">
         <div class="col general-info">
             <h3 class="fw-medium" [class.mb-0]="!courseDescription">{{ course.title }}</h3>
             <ng-container *ngIf="courseDescription">
                 <h5 class="fw-medium">{{ courseDescription }}</h5>
-                <h6 class="fw-medium" role="button" *ngIf="enableShowMore" (click)="toggleCourseDescription()">
+                <h6 class="fw-medium show-more" role="button" *ngIf="enableShowMore" (click)="toggleCourseDescription()">
```
Comment (ge65cer) [inline src/main/webapp/app/overview/header-course.component.html:7]: Could also be done with `mb-0`.

**Review by TheZoker — APPROVED** [review]
LGTM 👍

**Review by LiaLism — APPROVED** [review]
The course icon size looks good. Tested on ts1.

**Review by grochlitzzz — APPROVED** [review]
Looks better now. Tested on ts1

**Pull request merged**